src/components/Table/Table.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import {
+  Table as ChakraTable,
+  Thead as ChakraTHead,
+  Tbody as ChakraTbody,
+  Tr as ChakraTr,
+  Th as ChakraTh,
+  Td as ChakraTd,
+  TableCaption as ChakraTableCaption,
+  useMultiStyleConfig,
+} from "@chakra-ui/react";
+
+import generateUUID from "../../helpers/generateUUID";
+
+export interface TableProps {
+  /** Additional class name for the `Table` component. */
+  className?: string;
+  /** Array of string values used to populate the `Table` column headers. */
+  columnHeaders?: string[];
+  /** Hex value to set the background color of the column headers. */
+  columnHeadersBackgroundColor?: string;
+  /** Hex value to set the text color of the column headers. */
+  columnHeadersTextColor?: string;
+  /** ID that other components can cross reference for accessibility purposes. */
+  id?: string;
+  /** If true, the first cell of each row in the `Table` component will be visually styled as a header.  The default value is false */
+  useRowHeaders?: boolean;
+  /** If true, a border will be displayed between each row in the `Table` component.  The default value is false. */
```

Comment:
This says that the default value is `false` but on line 44, it is set to `true`.